src/export.h

```diff
@@ -0,0 +1,29 @@
+#ifndef EXPORT_H_
+#define EXPORT_H_
+
+#include <vector>
+#include <span>
+#include <cstdint>
+
+using std::uint8_t;
+using std::uint32_t;
+
+struct Rgba
+{
+	uint8_t r, g, b, a;
```

Comment:
These variables are uninitialized garbage; always initialize members with `{}`. Prefer not to use C-style multi-variable definition in one line.